test/unit/ns.html

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<div class="body" role="main">
+  <section id="namespace-Foo">
+    <span id="simple-ns"/>
+    <h1>Simple NS<a class="headerlink" href="#namespace-Foo" title="Permalink to this heading">&#xB6;</a></h1>
+    <dl class="php class">
+      <dt class="sig sig-object php" id="Foo\A">
+        <em class="property">
+          <span class="pre">class</span>
+        </em>
+        <span class="sig-prename descclassname">
+          <span class="pre">Foo\</span>
+        </span>
+        <span class="sig-name descname">
+          <span class="pre">A</span>
+        </span>
+        <a class="headerlink" href="#Foo\A" title="Permalink to this definition">&#xB6;</a>
+      </dt>
+      <dd/>
+    </dl>
+    <dl class="php method">
+      <dt class="sig sig-object php" id="Foo\A::simplify">
+        <span class="sig-prename descclassname">
+          <span class="pre">Foo\A::</span>
+        </span>
+        <span class="sig-name descname">
+          <span class="pre">simplify</span>
+        </span>
+        <span class="sig-paren">(</span>
+        <span class="sig-paren">)</span>
+        <a class="headerlink" href="#Foo\A::simplify" title="Permalink to this definition">&#xB6;</a>
+      </dt>
+      <dd/>
+    </dl>
+    <p>Cross linking:</p>
+    <div class="highlight-default notranslate">
+      <div class="highlight">
+        <pre><span/>- :php:meth:`A::simplify`
```

Comment:
@markstory can you please enable the CI? Also why this cross reference is not rendered as link?